crates/builder/src/graph/adj_list.rs

```diff
@@ -189,26 +329,52 @@ impl<NI: Idx, NV, EV> DirectedDegrees<NI> for DirectedALGraph<NI, NV, EV> {
 }
 
 impl<NI: Idx, NV> DirectedNeighbors<NI> for DirectedALGraph<NI, NV, ()> {
-    type NeighborsIterator<'a> = std::slice::Iter<'a, NI> where NV: 'a;
+    type NeighborsIterator<'a> = TargetsIter<'a, NI> where NV: 'a;
 
-    fn out_neighbors(&self, node: NI) -> Self::NeighborsIterator<'_> {
-        self.al_out.targets(node).iter()
+    fn out_neighbors<'slice>(&self, node: NI) -> Self::NeighborsIterator<'_> {
+        self.al_out.targets(node).into_iter()
```

Comment:
Where does `'slice` come into play here, doesn't seem to be used?

Reply:
It's a leftover, interesting that clippy did not complain